### PR TITLE
Converting SystemColors into InstanceDescriptors using Color Convertor

### PR DESCRIPTION
**Pull request by Anipik**
Fixes https://github.com/dotnet/corefx/issues/34252

We combined the check of the known colors and system colors into one check in .net Core.
But this created a bug as ```typeof(Color).GetProperty(c.Name);``` on system colors donot give the required output.

 we can do almost a similar thing i.e first try as it was a system color, if it returns null try for colors.

### CODE REVIEW

**Comment by safern** [comment]
> We combined the check of the known colors and system colors into one check in .net Core.

The main reason was that we added support for `SystemColors` into `KnownColors` (Added them into `ColorTable` but forgot to update the TypeConverter.

File: src/System.ComponentModel.TypeConverter/src/System/Drawing/ColorConverter.cs [hunk 1]
```diff
@@ -112,7 +112,7 @@ public override object ConvertTo(ITypeDescriptorContext context, CultureInfo cul
                     }
                     else if (ColorTable.IsKnownNamedColor(c.Name))
                     {
-                        member = typeof(Color).GetProperty(c.Name);
+                        member = typeof(SystemColors).GetProperty(c.Name) ?? typeof(Color).GetProperty(c.Name);
```
Comment (safern) [inline src/System.ComponentModel.TypeConverter/src/System/Drawing/ColorConverter.cs]: I would do the `Color` call first as those are the common Colors.

File: src/System.ComponentModel.TypeConverter/tests/Drawing/ColorConverterTests.cs [hunk 2]
```diff
@@ -446,5 +447,21 @@ public void GetStandardValuesExclusive()
...
+            Assert.Equal("Blue", conv.ConvertTo(Color.Blue, typeof(string)));
+            Assert.Equal("ActiveCaption", conv.ConvertTo(SystemColors.ActiveCaption, typeof(string)));
+        }
+
+        [Fact]
+        public void ConvertToInstanceDescriptorTests()
+        {
+            var conv = new ColorConverter();
+            Assert.NotNull(conv.ConvertTo(Color.Blue, typeof(InstanceDescriptor)));
+            Assert.NotNull(conv.ConvertTo(SystemColors.ActiveCaption, typeof(InstanceDescriptor)));
```
Comment (safern) [inline src/System.ComponentModel.TypeConverter/tests/Drawing/ColorConverterTests.cs]: Maybe we could invoke the property to get the value and compare it here to make sure we got the right property?

Comment (Anipik) [reply src/System.ComponentModel.TypeConverter/tests/Drawing/ColorConverterTests.cs]: how about something like this 
```c#
            InstanceDescriptor t = (InstanceDescriptor)conv.ConvertTo(SystemColors.ActiveCaption, typeof(InstanceDescriptor));
            Assert.Equal("ActiveCaption", t.MemberInfo.Name);
```

**Comment by Anipik** [comment]
The failure is in the helix job and is not related to the change

**Pull request merged**